src/Sdk/DTExpressions2/Expressions2/ExpressionConstants.cs

```diff
@@ -5,7 +5,7 @@
 
 namespace GitHub.DistributedTask.Expressions2
 {
-    internal static class ExpressionConstants
+    public static class ExpressionConstants
```

Comment:
lets not port the changes to this file to the server. we can revert this soon i think

Reply:
agree, that's what i planed. only on runner now and wait for your schema changes that has expression function built-in